src/posts/posts.service.ts

```diff
@@ -134,7 +144,8 @@ export class PostsService {
                 id
             }
         })
-       return deletedPost;
+        const action = await this.actionsService.addAction('Delete',userId,'Post',post.id,{post})
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them